vividus-plugin-lighthouse/src/main/java/org/vividus/lighthouse/LighthouseSteps.java

```diff
@@ -66,11 +90,18 @@ public class LighthouseSteps
     private final String apiKey;
     private final List<String> categories;
     private final int acceptableScorePercentageDelta;
+    private final PerformanceValidationStrategy performanceValidationStrategy;
 
     public LighthouseSteps(String applicationName, String apiKey, List<String> categories,
-            int acceptableScorePercentageDelta, IAttachmentPublisher attachmentPublisher, ISoftAssert softAssert)
+            int acceptableScorePercentageDelta, PerformanceValidationStrategy performanceValidationStrategy,
+            IAttachmentPublisher attachmentPublisher, ISoftAssert softAssert)
             throws GeneralSecurityException, IOException
     {
+        isTrue(performanceValidationStrategy != null, "The performance validation strategy should be set");
```

Comment:
why do users need to set performance validation strategy if they do not validate perforamnce?

Reply:
added more detailed message